src/main/modules/timeouts/index.ts

```diff
@@ -18,6 +19,10 @@ export class LoadTimeouts {
         errorController.internalServerError(err, req as AppRequest, res);
       });
 
+      if ((req.path as PageLink).includes(DOCUMENT_MANAGER)) {
+        timeoutMs = config.get<number>('uploadTimeout');
```

Comment:
Had to increase timeout to 60 seconds because 25MB upload can take longer than 30s